packages/demo-wallet-client/webpack.common.js

```diff
@@ -0,0 +1,160 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+const ESLintPlugin = require("eslint-webpack-plugin");
+const ProvidePlugin = require("webpack").ProvidePlugin;
+const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
+
+module.exports = {
+  entry: "./src/index.tsx",
+  mode: "development",
+  optimization: {
+    usedExports: true,
+    splitChunks: {
+      cacheGroups: {
+        source: {
+          test: /[\\/]src[\\/]/,
+          name: "source",
+          chunks: "all",
+        },
+        "vendor-react-redux": {
+          test: /[\\/]node_modules[\\/](react|react-dom|react-router|react-router-dom|react-redux|redux|redux-thunk|@reduxjs)[\\/]/,
+          name: "vendor-react-redux",
+          chunks: "all",
+        },
+        "vendor-stellar": {
+          test: /[\\/]node_modules[\\/](@stellar|stellar)[\\/]/,
+          name: "vendor-stellar",
+          chunks: "all",
+        },
+        "vendor-wallets": {
+          test: /[\\/]node_modules[\\/](@albedo|@ledgerhq|trezor)[\\/]/,
+          name: "vendor-wallets",
+          chunks: "all",
```

Comment:
I like this organization of node_modules! From the docs, my assumption this is a way to make sure we don't duplicate any of these libs (thus reducing bloat). Is that what your aim is here? Curious if there are any other benefits I'm missing

Reply:
If I remember correctly this is to make the generated file smaller by adding these only once. It was initially done in the Account Viewer in [this PR](https://github.com/stellar/account-viewer-v2/pull/387). Looks like it was a lot of trial and error to get it right. :sweat_smile: